Readme.md

```diff
@@ -34,28 +34,28 @@ Go dependencies are provided by the [Go machinery](https://nixos.org/nixpkgs/man
 
 For local development you may use `dep` to install go dependencies: `cd src/dividat-driver && dep ensure`.
 
-New Go dependencies can be added with `dep` (e.g. `dep ensure -add github.com/something/supercool`). The Nix specification of dependencies will recreated on subsequent builds (i.e. running `make`). Check in the updated `Gopkg.toml`, `Gopkg.lock` and `nix/deps.nix` files.
+New Go dependencies can be added with `dep` (e.g. `dep ensure -add github.com/something/supercool`). Make sure you run `make nix/deps.nix` to update the Nix expression containing go dependencies.
```

Comment:
This was actually changed here: https://github.com/dividat/driver/pull/97

This change simply updates the Readme to reflect status-quo.

> Would it be possible to make Gopkg.lock an input to the derivation?

This would be the nicest solution. However this requires being able to parse the Gopkg.lock from Nix.

I believe Rust people do things like that (they have a TOML parser in Nix: https://github.com/mozilla/nixpkgs-mozilla/blob/master/lib/parseTOML.nix).

Reply:
As Gopkg and Gopkg.lock also are written in TOML, take a few hours to look into reusing the TOML parser for that purpose. If not, we go ahead with the current setup and try to make it nicer/more obvious through tooling.